xhgui/xhgui.config.php

```diff
@@ -14,9 +14,9 @@
 
     // Database options for PDO.
     'pdo' => [
-        'dsn' => getenv('XHGUI_PDO_DSN') ?: null,
-        'user' => getenv('XHGUI_PDO_USER') ?: null,
-        'pass' => getenv('XHGUI_PDO_PASS') ?: null,
+        'dsn' => getenv('XHGUI_PDO_DSN') ?: 'mysql:host=db;dbname=xhgui',
```

Comment:
I like it.

Reply:
Not sure if it's better to nudge people to update `docker-compose.xhgui.yaml` or `xhgui.config.php`?
We pass environmental through docker-compose so these are DDEV-prefered fallbacks.

Reply:
I don't recommend having them touch either one. Why would we have them change any file? xhgui's files can create the db if it doesn't exist, and use the special db. The only problem, as you mentioned elsewhere, might be postgres. It doesn't know how to use postgres?